src/test/regress/expected/aggregate_support.out

```diff
@@ -1112,16 +1112,15 @@ SELECT create_distributed_table('dummy_tbl','a');
 create function dummy_fnc(a dummy_tbl, d double precision) RETURNS dummy_tbl
     AS $$SELECT 1;$$ LANGUAGE sql;
 -- test in tx block
--- shouldn't distribute, as citus.create_object_propagation is set to deferred
 BEGIN;
 create aggregate dependent_agg (float8) (stype=dummy_tbl, sfunc=dummy_fnc);
 COMMIT;
--- verify not distributed
+-- verify distributed
 SELECT run_command_on_workers($$select aggfnoid from pg_aggregate where aggfnoid::text like '%dependent_agg%';$$);
- run_command_on_workers
+               run_command_on_workers
 ---------------------------------------------------------------------
- (localhost,57637,t,"")
- (localhost,57638,t,"")
+ (localhost,57637,t,aggregate_support.dependent_agg)
+ (localhost,57638,t,aggregate_support.dependent_agg)
 (2 rows)
```

Comment:
This was testing the `deferred` behaviour without explicitly setting the setting. Now the `deferred` behaviour is not tested anymore.

Would it make sense to instead explicitly set the mode to `deferred`?

There is also a good reason to have a test for the default behaviour, but I think _not_ having a test for the deferred behaviour might cause regressions overtime. Either with this object or others.